pkg/operator/promsum.go

```
@@ -301,7 +301,7 @@ func (op *Reporting) newPromImporterCfg(reportDataSource *cbTypes.ReportDataSour
 	// it would take to chunk up our MaxQueryRangeDuration.
 	defaultMaxPromTimeRanges := int64(op.cfg.PrometheusDataSourceMaxQueryRangeDuration / chunkSize)
 
-	tableName := reportingutil.FullyQualifiedTableName(prestoTable)
+	tableName, _ := reportingutil.FullyQualifiedTableName(prestoTable)
```

Don't ignore the err

---

Updated all references.